src/api.tf

```diff
@@ -440,7 +440,6 @@ module "rtd_csv_transaction" {
   api_management_name = module.apim.name
```

Comment:
Maybe we need a `count` meta-argument to exclude production environment. I think it's preferable to harden the blob storage before moving to production. What's your opinion?

Reply:
Agree. I'll add a commit to include this feature.